.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.19
```

Comment:
In the `go.mod` file we are using `1.12` currently.
Can you either update this here to also use `1.12` (not sure this is still supported by this action) or update the go version in the `go.mod` file?

Reply:
With go 1.12 tests was failed, so I've bumped version, because test package want's 1.20.
I'll try to update go version to 1.20